tests/test_kubectl.py

```diff
@@ -0,0 +1,33 @@
+
+import pytest
+from pytest_container import DerivedContainer
+from pytest_container.container import ContainerLauncher
+from pytest_container.runtime import OciRuntimeBase
+
+from bci_tester.data import KUBECTL_CONTAINERS
+
+
+@pytest.mark.parametrize("ctr_image", KUBECTL_CONTAINERS)
+def test_kubectl_binary(
+    container_runtime: OciRuntimeBase,
+    pytestconfig: pytest.Config,
+    ctr_image: DerivedContainer,
+) -> None:
+    kubectl_ctr = DerivedContainer(
+        base=ctr_image,
+        custom_entry_point="/bin/bash",
+    )
+
+    with ContainerLauncher.from_pytestconfig(
+        kubectl_ctr, container_runtime, pytestconfig
+    ) as kubectl_launcher:
+        kubectl_launcher.launch_container()
+        kubectl_con = kubectl_launcher.container_data.connection
+        output = kubectl_con.check_output(
+            "kubectl create deployment nginx --image=nginx --dry-run=client -o yaml"
```

Comment:
Can you output json? If yes, then do that and load it with python and deep compare the actual result. We don't really care about the *exact* output of the command, but more about the contents.